packages/stream_chat/lib/src/core/models/filter.dart

```diff
@@ -43,6 +43,9 @@ enum FilterOperator {
   /// Matches values that exist/don't exist based on the specified boolean value.
   exists,
 
+  /// Matches values that don't exist.
+  notExists,
+
```

Comment:
we don't really need this

Reply:
Yeah fair enough, it was done in another way before, it changed to make it non breaking